user-stories/US-08-Data-Consumer-interface.md

```diff
@@ -0,0 +1,185 @@
+
+## US-08-01
+
+**As a** Data Consumer,
+
+**I want to** be able to see the list of all Privacy requests (new ones and processed one) I received
+
+**because** I need to keep track of them for legal reason
+
+## US-08-02
+
+**As a** Data Consumer,
+
+**I want to** be able to be informed with a notification on the tab label when I receive a new Privacy request, and that this notification mark stays until I have answered all requests
+
+**because** I want to easily see I have new/pending Privacy requests so I don’t forget to answers them and stay in legality
+
+## US-08-03
+
+**As a** Data Consumer,
+
+**I want to** be able to choose how I want to organize the order of my list of Privacy requests according to the date : either the most recent one top to first see the ones that I just received or the most old ones to first see the ones that require to be process first 
+
+**because** I need to organize the list in way that it is easier for me to process all Privacy requests in legal due time
+
+## US-08-04
+
+**As a** Data Consumer,
+
+**I want to** clearly see the deadline by which I have to legally reply to a Privacy request 
+
+**because** I want to stay out of any legal trouble and process all the Privacy requests I receveice in the legal due time
+
+## US-08-05
+
+**As a** Data Consumer,
+
+**I want to** view the details of each Privacy request I received (date of the request, Submitter identification, type of request and details of request (data category, processing catgory, target, provenance...), additional elements if there are any such as identification document, message, data range, data capture, data reference, selected data to delete, selected data to modify, new data modified, consent restriction...)
+
+**because** I need to take notice of them to proccess the Privact request
+
+## US-08-06
+
+**As a** Data Consumer,
+
+**I want to** be able to see, download and print the details of old Privacy Requests, a proof of compliance for treated Privacy Requests, with the following information : (date of the request, Submitter identification, type of request and details of request (data category, processing catgory, target, provenance...), additional elements if there are any such as identification document, message, data range, data capture, data reference, selected data to delete, selected data to modify, new data modified, consent restriction...), type of answer (approved/partially approvec/denied), date of the answer, date of the action associated with the type of request (e.g user has accessed data)
+
+**because** I need to keep track of them for legal reason or I might need to give to my boss or colleague
+
+## US-08-07
+
+**As a** Data Consumer,
+
+**I want to** process the Privacy requests, that can't be automatically processed, directly from the interface
+
+**because** it is not convenient for me to manually write all an email and manually fetch all elements to answer the Privacy request
```

Comment:
```suggestion
**because** it is not convenient for me to manually write an email and manually fetch all elements to answer the Privacy request
```

Reply:
Here also - I might see a recommended anser, but I might want to modify it, choose a different one.

Reply:
I updated : 
## US-08-10

**As a** Data Consumer,

**I want to** be guided to answer a Privacy Request, for example having a recommendation that I can modify or a template if I need to add a message to specify why I must deny a Privacy Request

**because** I want to save time and I might not know what can I answer or what are the valid reasons to deny a Privacy request